frontend/src/i18/english.ts

```diff
@@ -419,6 +419,7 @@ export const english = {
     'Client shares score': 'Client shares score',
     Poor: 'Poor',
     Great: 'Great',
-    'Reset taskgroup positions': 'Reset taskgroup positions',
+    'Fit view -tooltip': 'Fit taskgroups to the current viewport',
+    'Taskgroup reset -tooltip': 'Reset task group positions',
```

Comment:
The new tooltip msg is a little long, could the following work?
```suggestion
    'Fit view -tooltip': 'Fit groups to the viewport',
    'Taskgroup reset -tooltip': 'Reset group positions',
```

Reply:
Yeah these seem better! I'll do the change